packages/ui-react/src/components/TileDock/TileDock.tsx

```diff
@@ -235,8 +235,9 @@ function TileDock<T>({
     if (showDots && isMultiPage && !!renderPaginationDots) {
       const length = pages;
 
+      // Adding role and aria-hidden to the container to avoid screen readers from reading the dots as these is a purely visual indicator
       return (
-        <div className={styles.dots}>
+        <div aria-hidden role="presentation" className={styles.dots}>
```

Comment:
We should apply `aria-hidden="true"`. `aria-hidden` without without an attribute has no purpose and combining it with role="presentation" is weird. It's one or the other.

And the comment should indicate that this is not the ideal solution, but something to improve upon (due to virtualization challenges). Currently the comment states what the `aria-hidden="true"`  / `role="presentation"` purpose is, which has no additional value.

Reply:
I've removed role presentation in favor of aria-hidden. The comment now reflects de thoughts behind the prop a bit better.